sys/net/gnrc/pktbuf_static/gnrc_pktbuf_static.c

```diff
@@ -82,9 +86,9 @@ static inline void _set_pktsnip(gnrc_pktsnip_t *pkt, gnrc_pktsnip_t *next,
 void gnrc_pktbuf_init(void)
 {
     mutex_lock(&_mutex);
-    _first_unused = (_unused_t *)_pktbuf;
+    _first_unused = (_unused_t *)_pktbuf_buf;
```

Comment:
This is where the alignment previously was not handled correctly. It was just casting a char array to an `unused_t` pointer.

Reply:
As `_pktbuf` was in static memory, this seems to be to me more of a false positive. But if it makes the compiler happier, I'm not blocking.

Reply:
### `test.c`
```C
#include <stdio.h>

static char a[3];
static char b[3];
static char c[3];

int main(void)
{
    printf("address of a = %p\n", (void *)a);
    printf("address of b = %p\n", (void *)b);
    printf("address of c = %p\n", (void *)c);

    return 0;
}
```

```
$ gcc -Wall -Wextra -pedantic -std=c99 -Os -o test test.c 
$ ./test
address of a = 0x5653e3786076
address of b = 0x5653e3786073
address of c = 0x5653e3786070
```